identity-and-access-management/secret-manager/how-to/filter-secrets.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: How to filter your secrets
+  description: This page explains how to filter your secrets
+content:
+  h1: How to filter your secrets
+  paragraph: This page explains how to filter your secrets
+tags: secret-manager secret
+dates:
+  validation: 2023-02-15
+  posted: 2023-02-15
+categories:
+  - identity-and-access-management
+---
+
+This page explains how to filter your [secrets](/identity-and-access-management/secret-manager/concepts/#secret/) using the [Scaleway console](https://console.scaleway.com).
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [generated an API key](/identity-and-access-management/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/identity-and-access-management/iam/reference-content/permission-sets/)
+  - Your have [several secrets](/identity-and-access-management/secret-manager/how-to/create-secret/)
+</Message>
+
+## Filter your secrets by tags
```

Comment:
On ne met plus les "How-to" dans les titres ?

Reply:
On les met quand y'a deux sortes de how-tos dans un seul, non ? Genre là y a "filter by tags" et juste après "filter by region"